src/metalperformanceshaders.cs

```diff
@@ -1135,5 +1525,1898 @@ interface MPSMatrixMultiplication {
+	[BaseType (typeof(MPSImageArithmetic))]
+	interface MPSImageMultiply
+	{
+		[Export ("initWithDevice:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (IMTLDevice device);
+	}
+
+	[TV (11,0), Mac (10, 13, onlyOn64: true), iOS (11,0)]
+	[BaseType (typeof(MPSImageArithmetic))]
+	interface MPSImageDivide
+	{
+		[Export ("initWithDevice:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (IMTLDevice device);
+	}
+
+	[TV (11,0), Mac (10, 13, onlyOn64: true), iOS (11,0)]
+	[BaseType (typeof(MPSUnaryImageKernel))]
+	interface MPSImageScale
+	{
+		[Export ("initWithDevice:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (IMTLDevice device);
+
+		[NullAllowed, Export ("scaleTransform", ArgumentSemantic.Assign)]
+		MPSScaleTransform ScaleTransform { get; set; }
+
+		[Export ("initWithCoder:device:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSCoder aDecoder, IMTLDevice device);
+	}
+
+	[TV (11,0), Mac (10, 13, onlyOn64: true), iOS (11,0)]
+	[BaseType (typeof(MPSImageScale))]
+	interface MPSImageBilinearScale
+	{
+		[Export ("initWithDevice:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (IMTLDevice device);
+
+		[Export ("initWithCoder:device:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSCoder aDecoder, IMTLDevice device);
+	}
+
+	[TV (11,0), Mac (10, 13, onlyOn64: true), iOS (11,0)]
+	[BaseType (typeof(MPSUnaryImageKernel))]
+	interface MPSImageStatisticsMinAndMax
+	{
+		[Export ("clipRectSource", ArgumentSemantic.Assign)]
+		MTLRegion ClipRectSource { get; set; }
+
+		[Export ("initWithDevice:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (IMTLDevice device);
+
+		[Export ("initWithCoder:device:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSCoder aDecoder, IMTLDevice device);
+	}
+
+	[TV (11,0), Mac (10, 13, onlyOn64: true), iOS (11,0)]
+	[BaseType (typeof(MPSUnaryImageKernel))]
+	interface MPSImageStatisticsMeanAndVariance
+	{
+		[Export ("clipRectSource", ArgumentSemantic.Assign)]
+		MTLRegion ClipRectSource { get; set; }
+
+		[Export ("initWithDevice:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (IMTLDevice device);
+
+		[Export ("initWithCoder:device:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSCoder aDecoder, IMTLDevice device);
+	}
+
+	[TV (11,0), Mac (10, 13, onlyOn64: true), iOS (11,0)]
+	[BaseType (typeof(MPSUnaryImageKernel))]
+	interface MPSImageStatisticsMean
+	{
+		[Export ("clipRectSource", ArgumentSemantic.Assign)]
+		MTLRegion ClipRectSource { get; set; }
+
+		[Export ("initWithDevice:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (IMTLDevice device);
+
+		[Export ("initWithCoder:device:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSCoder aDecoder, IMTLDevice device);
+	}
+
+	[TV (11,0), Mac (10, 13, onlyOn64: true), iOS (11,0)]
+	[BaseType (typeof(NSObject), Name = "MPSNNDefaultPadding")]
+	interface MPSNnDefaultPadding : MPSNnPadding
+	{
+		[Static]
+		[Export ("paddingWithMethod:")]
+		MPSNnDefaultPadding PaddingWithMethod (MPSNnPaddingMethod method);
```

Comment:
Methods should have verbs, so maybe `CreatePaddingWithMethod`?

Reply:
Probably just CreatePadding